_episodes/44.md

```diff
@@ -0,0 +1,213 @@
+
+[Trino 405]({{site.url}}/docs/current/release/release-405.html)
+
+* Support for `ALTER COLUMN ... SET DATA TYPE` statement.
+* Support for Apache Arrow when reading from BigQuery.
+* Support for views in the Delta Lake connector.
+* Support for the Iceberg REST catalog.
+* Support for Protobuf encoding in the Kafka connector.
+* Support for fault-tolerant execution in the MongoDB connector.
+* Support for `DELETE` and query pushdown in the Redshift connector.
+* Performance improvements when reading Parquet data.
+
+[Trino 406]({{site.url}}/docs/current/release/release-406.html)
+
+* tbd
+
+Further thoughts and highlights from Cole:
+
+* tbd
+
+More detailed information is available in the release notes for
+[Trino 405]({{site.url}}/docs/current/release/release-405.html),
+and
+[Trino 406]({{site.url}}/docs/current/release/release-406.html).
+
+We also shipped trino-python-client 0.321.0 with the following improvements:
+
+* Add support for SQLAlchemy 2.0.
+* Add support for `varbinary` query parameters.
+* Add support for variable precision `datetime` types.
+
+## What is Metabase
+
+[Metabase](https://www.metabase.com/) is the BI tool with the friendly UX and
+integrated tooling to let your company explore data on their own.
+
+Core features and advantages of Metabase include the following:
+
+* Visual query build
+* Dashboards
+* Models
+
+Metabase is a web-based application that you run on a server and make it
+available for multiple users. It uses SQL to create queries, reports,
+visualizations, dashboards and more.
+
+Metabase is an open source project licensed under the GNU Affero General Public
+License (AGPL) license. It is written in Clojure and therefore runs on the Java
+virtual machine.
+
+Metabase is also the name of the company, founded in 20xx. It provides an
+expanded version under a commercial license, a SaaS version of the application,
+support and others services, and manages the open source project.
+
+## History of Metabase and Trino
+
+Metabase was first released in 2015 as version 0.9. Since the initial release it
+has grown to be a well known and widely used BI application.
+
+A Presto driver was created in 2018. It directly integrated with the client REST
+API. With the rename of Presto to Trino, Manfred [created a
+PR](https://github.com/metabase/metabase/pull/15160) that replicates this for
+Trino to ensure continued support for the community. In the discussion it was
+decided that it would be better to use the Trino JDBC driver, similar to how
+other drivers for Metabase work.
+
+After some more demand from the user and customer community, Starburst and
+Metabase established a collaboration, and started implementation of the current
+driver. Piotr led the charge, Andrew buckled down and learned Clojure, and
+together a first release was created and tested. The driver is now provided as
+an open source project managed by Starburst.
+
+## Core advantages of using Metabase with Trino
```

Comment:
This section should be mostly covered by @paoliniluis and @leniartek .. with moderation and questions from @colebow and myself